src/index.js

```diff
@@ -1,24 +1,30 @@
-'use strict';
+import { JSDOM } from 'jsdom';
 
-import dom from 'jsdom';
+const SGPC_URL = 'http://old.sgpc.net/hukumnama/sgpconlinehukamnama.asp';
 
-const SGPC_URL = `http://old.sgpc.net/hukumnama/sgpconlinehukamnama.asp`;
+module.exports = () => new Promise((resolve, reject) => {
```

Comment:
We can use `export default`

Reply:
Depending on if we use Babel, and if we're targeting Node 8